### PR TITLE
Use createGroupAndAddAuth when inviting external agents

### PR DESCRIPTION
**Pull request by monitz87**
### Description

- Replace the use of `createGroup` in the `inviteExternalAgent` procedure with `createGroupAndAddAuth`

### JIRA Link

https://polymath.atlassian.net/browse/DA-353

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/utils/__tests__/conversion.ts [hunk 1]
```diff
@@ -3236,7 +3236,7 @@ describe('claimToMeshClaim and meshClaimToClaim', () => {
       let value: Claim = {
         type: ClaimType.Jurisdiction,
         code: CountryCode.Cl,
-        scope: { type: ScopeType.Identity, value: 'someTickerDid' },
+        scope: { type: ScopeType.Identity, value: 'SOME_TICKERDid' },
```
Comment (polymath-eric) [inline src/utils/__tests__/conversion.ts:3239]: (nit) mixed case looks weird to me

```suggestion
        scope: { type: ScopeType.Identity, value: 'SOME_TICKER_DID' },
```

File: src/api/procedures/inviteExternalAgent.ts [hunk 2]
```diff
@@ -114,16 +130,22 @@ export async function prepareInviteExternalAgent(
...
-    })) as PostTransactionValue<CustomPermissionGroup>;
-    postTransactionAuthorization = createGroupResult.transform(createBecomeAgentData);
-
-    rawAuthorizationData = postTransactionAuthorization.transform(authorization =>
-      authorizationToAuthorizationData(authorization, context)
-    );
+    const rawTicker = stringToTicker(ticker, context);
+    const { transactions } = permissionsLikeToPermissions(permissions, context);
+
+    await assertGroupDoesNotExist(asset, transactions);
```
[the remark this answers is not in the document]
Comment (monitz87) [reply src/api/procedures/inviteExternalAgent.ts:136]: Now that I think about it, it does make sense. I guess I thought that if users were passing permissions explicitly, they were expecting a group to be created, but we're actually "hiding" that from them and it shouldn't make much of a difference. Nice catch

File: src/utils/__tests__/conversion.ts (hunk 1 above)
Comment (monitz87) [reply src/utils/__tests__/conversion.ts]: My bad, I did a mass replace of `someTicker` to `SOME_TICKER`

**Review by VictorVicente — APPROVED** [review]

File: src/utils/__tests__/conversion.ts (hunk 1 above)
Comment (polymath-eric) [reply src/utils/__tests__/conversion.ts]: There's a few more spots like this still

**Pull request merged**